code/modules/mob/living/say.dm

```diff
@@ -256,6 +285,10 @@ var/list/department_radio_keys = list(
 
 	if(stuttering)
 		message = stutter(message)
+		
+	//languages
+		message = "<span class='name'>[speaker]</span> squawks incoherently."
```

Comment:
1. Why is this under a stuttering check?
2. This just sets the message to squawks incoherently.